zerver/openapi/zulip.yaml

```diff
@@ -3228,6 +3228,12 @@ paths:
                                             [setting-values]: /api/group-setting-values
                                             [system-groups]: /api/group-setting-values#system-groups
                                             [mentions]: /help/mention-a-user-or-group
+                                    deactivated:
+                                      type: boolean
+                                      description: |
+                                        Whether the user group is deactivated or not.
```

Comment:
Probably we should say a bit more about what this means. "Whether the user group is deactivated. Deactivated groups cannot be used as a subgroup of another group or used for any other purpose."

Reply:
Updated.